docs/admin/upgrading.rst

```diff
@@ -37,6 +37,15 @@ Basic instructions
    Any modifications should have been written within plugins, so that they are not lost on overwriting.
    If this is not the case, take care to maintain your modifications. 
 
+From 2.x to 3.0
```

Comment:
Sorry to nitpick, but I'd rather be explicit about "2.3 to 3.0." I don't want to give the impression that site owners can just skip the minor release upgrades and notes.